avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1215,7 +1224,12 @@ private Schema toConnectSchema(org.apache.avro.Schema schema, boolean forceOptio
         throw new DataException("Invalid schema name: " + connectNameJson.toString());
       }
       name = connectNameJson.asText();
-
+    } else if (schema.getLogicalType() != null) {
+      String logicalTypeName = schema.getLogicalType().getName();
+      name = AVRO_LOGICAL_TYPE_NAMES_TO_CONNECT_LOGICAL_TYPE_NAMES.get(logicalTypeName);
```

Comment:
What are the compatibility concerns here? It seems like we're going to capture names for Avro logical types that aren't directly supported by Connect (e.g. `time-micros`, `timestamp-micros`, `duration`). Since this is the Avro -> Connect conversion, I'm not sure this is necessarily an issue, but strictly speaking the schema will be different so existing users of those types would observe a change in schema when updating to this new version. None of the names would *change* as far as I can tell since all the Avro logical types are primitives, but they would have a name added.